hicommonwealth/commonwealth
# Thread lock redesign

## Link to Issue
Closes: #3931

## Description of Changes
- Shows new design for locked thread, which includes date when locked

## Test Plan
- Run migration locally
- Go to a previously locked thread– confirm that it shows the "locked prior to" text (null locked timestamp)
- Unlock, then lock the thread again– confirm that it shows the "locked at" text (not-null locked timestamp)

## Deployment Plan
- Requires running migration to add `locked_at` column to `Threads` table (not sure if this happens automatically or not during deploy)

## Other Considerations
N/A
